user_app/bin/model/course.dart

```diff
@@ -0,0 +1,7 @@
+import '../utils/config.dart';
+/// The available courses for the user
+/// The no of courses to enroll can be specifed thought [Config.minCourseEntry]
+/// and [Config.maxCourseEntry]
+enum Course {
+  A, B, C, D, E, F
+}
```

Comment:
Read about extensions when you are using enums

Reply:
Where to use them?

Reply:
Just to go through and understand. No use-case for now 
In previous dart versions there were no .values.byName() function 
In that case you could have written an extension that contained a function like this

Reply:
Okay got it.